api/src/database/seeds/02-roles.yaml

```diff
@@ -23,8 +23,6 @@ columns:
     default: false
   module_list:
     type: json
-  collection_list:
```

Comment:
We shouldn't touch the seeds at all, otherwise existing installations will get out of sync with new installations